packages/@okta/vuepress-site/docs/reference/scim/scim-20/index.md

```diff
@@ -10,7 +10,7 @@ meta:
 
 This reference focuses on how Okta API endpoints share information with System for Cross-domain Identity Management (SCIM) specific API calls. This document specifically covers Version 2.0 of the SCIM specification.
```

Comment:
"This document specifically covers Version 2.0 of the SCIM specification" -- this sentence is repeated.